### ISSUE TITLE
Under active maintenance?

### ISSUE THREAD

**Issue by sebastiankade** [facebookexperimental/Recoil#2288]
Things seem eerily quite for the last 6 months, is Recoil still being actively developed?

**Comment by mbrimmer83** [comment]
Some minor changes were committed a week ago. I really hope they continue maintaining it. I love it so much more than Jotai tbh.

**Comment by SujayDas1999** [comment]
I am new to recoil and so far i am absolutely loving it! I hope they regularly maintain it <3

**Comment by ndobbs** [comment]
Been some time without an update and there are a few bugs tucked away in corners, especially around Duplicate Atom Key warnings/detection etc. I wrote an eslint script to try to find them but they don't exist :ghost: 

So we've disabled the checking and rely only on the eslint script for dupe detection.

**Comment by oadrian2** [comment]
We just hit month 8 without an update. Any updates? I understand that the layoffs probably messed up stuff over on the RecoilJS team and I know at least one major contributor left the company. At this point, I feel like leaving this up here in an unknown state is doing reputational damage to Facebook. They could either wave people off and say that support is uncertain at this point or make any statement doing even a soft commit to continuing work on RecoilJS.

**Comment by mildfuzz** [comment]
Any update on this? Should we be looking at alternatives at this point?

**Comment by dpalay** [comment]
> Any update on this? Should we be looking at alternatives at this point?

I'm looking at Preact's Signals-react. It has a very Recoil feel to it.

**Comment by Lc0rE** [comment]
Any news from the dev team? Should we start looking for alternatives?

**Comment by clockelliptic** [comment]
Hi everyone 👋,

TLDR; Check out [clockelliptic/jotai-recoil-adapter](https://github.com/clockelliptic/jotai-recoil-adapter). It's rough around the edges and could use community support, but has proven to be very useful so far. Please feel invited to contribute, fork, and copy as needed.

see it on NPM: https://www.npmjs.com/package/jotai-recoil-adapter

see also: #1495 

🔍 What is jotai-recoil-adapter?
The `jotai-recoil-adapter` is intended to facilitate a less-painful transition from Recoil to Jotai. It provides an API compatible with Recoil while leveraging the simplicity and efficiency of Jotai under the hood. Our goal is to make it easier for teams reliant on Recoil to migrate to Jotai without having to overhaul their existing codebases.

🌟 Features Include:
- An API mirroring key Recoil functionalities, such as `atom`, `selector`, `useRecoilState`, and more.

🤝 We Need Your Input!
This project is in its early stages, and community input is invaluable.

Current top priorities include:
 - expanded adapter APIs to support migration from Recoil's writable selectors (proxy selectors)
 - expanded adapter APIs for other Recoil APIs
 - test suite development
 - development of migration docs for Recoil APIs that aren't adaptable to Jotai

🔗 Get Involved:
- Check out the repository: [clockelliptic/jotai-recoil-adapter](https://github.com/clockelliptic/jotai-recoil-adapter)
- Try it out in your projects and let us know your experience.
- Report any issues or suggest enhancements on our issues page.
- Feel free to fork the repo, submit PRs, or discuss potential features.

**Comment by kibertoad** [comment]
It's dead, Jim: https://twitter.com/dai_shi/status/1757965668425814380?s=19

**Comment by alxclark** [comment]
@pieterv @kassens @SamChou19815, sorry to ping you all like this, but you were both active contributors to the project. Do you know if there's going to be active support going forward, especially with the next major version of React coming up ? I'm maintaining a few projects using Recoil and it would be great to know if the recommendation is to plan a transition away from the project.

**Comment by kibertoad** [comment]
@alxclark Facebook itself has migrated away from Recoil: https://github.com/facebook/sapling/commit/547b205eab16fc78d73ec8edb38b2b2bdc84ddf2

So I think that answers your question.

**Comment by alxclark** [comment]
I'm aware of the comment right above mine, but it would be great to share openly the rationale behind the migration away and also if they foresee any major problems with the next version of React. 

I'm ok consuming software that is stable and solve my needs, even if not maintained or used by Meta, but if there's known issues and I will be forced to migrate in the next year to keep up with React, I prefer knowing and being able to plan for it.

**Comment by SamChou19815** [comment]
@alxclark This is not actively maintained. I only touched the code to upgrade Flow internally. We have since stopped the sync between internal and external repo for this project. (There is no further internal activity either). I am not the original author so I wouldn't recommend anything, but I guess you can already see the writing on the wall.

**Comment by manyopensource** [comment]
[Jotai](https://jotai.org/) is replacing Recoil.

**Comment by mildfuzz** [comment]
> [Jotai](https://jotai.org/) is replacing Recoil.

Is this official?

**Comment by manyopensource** [comment]
> Is this official?

No, it's not. I'm just following the Recoil for so long, and it seems like it's stuck for some reason. Maybe it is necessary, maybe not. But Jotai seems to have at least main the features of Recoil, and is being supported.
